winui/AvatarView/content-types.md

```diff
@@ -7,4 +7,305 @@ control: AvatarView
 documentation: ug
 ---
 
```

Comment:
@MadhumithanJayapathi Learn about the available content types and how to set values for them in the WinUI AvatarView control.

Reply:
@Manivannan-E  Changed the description as mentioned.